docker/z-quantum-default/Dockerfile

```diff
@@ -23,19 +26,22 @@ RUN apt-get -y install \
                 libzmq3-dev \
                 libz-dev \
                 libblas-dev \
-                liblapack-dev
+                liblapack-dev \
+                openssh-client
 
 
-# ZQuantum requires Python 3.7. By installing `python3-pip` we have also installed 
-# Python 3.8. We need to set back the Python version to 3.7.
-RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 1 && \
-    update-alternatives --set python3 /usr/bin/python3.7
+#- # After March 2022, git changed its authentication.
+#- RUN git config --global url."git@github.com:".insteadOf git://github.com/
 
-ENV PYTHONPATH="/usr/local/lib/python3.7/dist-packages:${PYTHONPATH}"
+# ZQuantum requires Python 3.8. By installing `python3-pip` we have also installed
```

Comment:
Is there a typo here? Do you mean that `python3-pip` installed Python 3.9 or the comment is obsolete and needs to be deleted?

Reply:
Will remove..